### ISSUE TITLE
Make a copy of wheel file in task-collection

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#1881]
When a user provides the path to a wheel file, it could be e.g. `/random/dir/fractal_tasks_mock-0.0.1-py3-none-any.whl`.
Before the actual `pip install` operation, we should
1. Copy the wheel file to the actual task-collection folder (e.g. `/somewhere/{user_id}/fractal-tasks-mock/0.0.1/`).
2. Update the corresponding `TaskGroupV2.wheel_path`.
3. pip install from there.

This way, the pip-freeze will include data like this
```
fractal-tasks-mock @ file:///somewhere/{user_id}/fractal-tasks-mock/0.0.1//fractal_tasks_mock-0.0.1-py3-none-any.whl
pydantic==1.10.18
typing_extensions==4.12.2
```
which will be reproducible even after deactivation.

---

TLDR: we cannot rely on the user-provided wheel to still exist, when we'll have to re-activate the venv.
